### ISSUE TITLE
Training on CIFAR10-DVS

### ISSUE THREAD

**Issue by FranziSchwaiger** [bamsumit/slayerPytorch#62]
Hi!

I am wondering whether anyone has tried to train a SNN with SLAYER on the CIFAR10-DVS dataset so far.
I am currently struggling to get my network generalizing, although I tried several things like dropout and data augmentation.

My network gets about 100% accuracy on the training set but only 20% on the test set.

I also tried to vary the neuron config parameters, but without any luck.

My current settings are: 

Simulation time 1000ms with a time step of 5ms (like it was also done in other papers).

My current network architecture:

<img width="198" alt="networks" src="https://user-images.githubusercontent.com/69162944/120065541-1705e880-c072-11eb-93c2-41eb4783219c.PNG">

My current neuron config:

<img width="54" alt="neuronConfig" src="https://user-images.githubusercontent.com/69162944/120065542-179e7f00-c072-11eb-821f-3c612912bffc.PNG">

For the loss I use the same as in NMNIST (60 for correct and 10 for incorrect predfictions)

Did you already try to train on this dataset and if yes, did you also run in this problem?

Thank you very much in advance!

**Comment by bamsumit** [comment]
Hi @FranziSchwaiger, I don't recall anyone who has tried CIFAR-10 DVS dataset. 

From what you mention, you seem to be having serious overfitting. But it's hard to think that the network you are using is overfitting. I would suggest looking at spikes in each intermediate layer and try to understand what's happening. Perhaps look at dataloader input for both training and testing before that as well.

**Comment by bamsumit** [comment]
Haven't heard back. Closing this issue.

**Comment by FranziSchwaiger** [comment]
Sorry for not responding, I haven't worked on this for a while.

Thank you very much, looking at the output spikes helped to solve the problem.

The solution was that I needed to scale the amount of spikes for the correct and incorrect class by the simulation step. 

The performance of the test set is now reasonable.

As architecture, I used the same as for the DVS-Gesture dataset but with doubled filters and with augmented data.
